src/Integration/Connection/BasicAuthenticationCredentialsValidator.cs

```diff
@@ -18,25 +18,20 @@ internal class BasicAuthenticationCredentialsValidator
     {
         private readonly static Regex InvalidCharacters = new Regex("[:]"); // colon
 
-        private string username;
-        private SecureString password;
+        private string user;
+        private SecureString pwd;
```

Comment:
Unrelated - renamed to fix code analysis issues (complaining about parameters and fields with the same names)